module/module_interceptors_test.go

```diff
@@ -188,6 +188,8 @@ func TestModuleClientTimeoutInterceptor(t *testing.T) {
 		_, err = helper1.DoCommand(ctxWithDeadline, map[string]interface{}{"command": "echo"})
 		test.That(t, err, test.ShouldBeNil)
 	})
+
+	test.That(t, r.Close(ctx), test.ShouldBeNil)
```

Comment:
[nit] Can you put this in a `defer func()` right after the call to `robotimpl.New()` so it runs even if one of the assertions above fails? Apologies, I wrote that robot creation code and forgot the corresponding `Close` 😮‍💨 .

Reply:
agree that this should be `defer`ed.

not necessarily something to do now, but it might be worth defining [a helper like this one](https://github.com/viamrobotics/rdk/blob/644a8952839aaa2a47e9a5a6aa278e03a5beb6e0/robot/impl/utils.go#L14) in the `module` package if this kind of local robot setup/close is a prevalent.

Reply:
`defer`ed 🫡